Xamarin.Forms.Core/Page.cs

```diff
@@ -190,17 +190,33 @@ public Task<string> DisplayActionSheet(string title, string cancel, string destr
 			return args.Result.Task;
 		}
 
-		public Task DisplayAlert(string title, string message, string cancel)
+		public Task<string> DisplayActionSheet(string title, string cancel, string destruction, FlowDirection flowDirection, params string[] buttons)
 		{
-			return DisplayAlert(title, message, null, cancel);
+			var args = new ActionSheetArguments(title, cancel, destruction, buttons);
+
+			args.FlowDirection = flowDirection;
+
+			if (IsPlatformEnabled)
+				MessagingCenter.Send(this, ActionSheetSignalName, args);
+			else
+				_pendingActions.Add(() => MessagingCenter.Send(this, ActionSheetSignalName, args));
+
+			return args.Result.Task;
+		}
+
+		public Task DisplayAlert(string title, string message, string cancel, FlowDirection flowDirection = FlowDirection.MatchParent)
```

Comment:
The compiler will spit this out as 
`public Task DisplayAlert(string title, string message, string cancel, FlowDirection flowDirection)`

So in order to not break people you have to retain the version without the extra parameter

```C#
public Task DisplayAlert(string title, string message, string cancel)
```

Otherwise libraries compiled against Xamarin.Forms will crash with a "Method Not Found Exception"


Same comment about DisplayActionSheet